packages/rocketchat-livechat/client/route.js

```diff
@@ -19,6 +19,22 @@ AccountBox.addRoute({
 	pageTemplate: 'livechatCurrentChats'
 }, livechatManagerRoutes);
 
+AccountBox.addRoute({
+	name: 'livechat-analytics',
+	path: '/livechat-analytics',
+	sideNav: 'livechatFlex',
+	i18nPageTitle: 'Analytics',
+	pageTemplate: 'livechatAnalytics'
+}, livechatManagerRoutes);
+
+AccountBox.addRoute({
+	name: 'livechat-real-time-monitoring',
+	path: '/livechat-real-time-monitoring',
```

Comment:
Let's remove the prefix `livechat` in `path` prop, like this -> `path: '/real-time-monitoring'`.